assets/ak-v3.css

```diff
@@ -5747,3 +5747,218 @@ a.bg-midgray:hover {
   color: white;
 }
 
+/* Add styles for larger radio buttons/check boxes */
```

Comment:
Looks like we're moving this from user_form_wrapper to CSS, and therefore all pages. Is that desired, or do we run risk of messing with pages we didn't want to mess with?

Reply:
this was intentional so that all pages use the larger checkbox/radio button styles, since we want them to be default now

Reply:
I think we might still be waiting on QA for this one though? https://github.com/350org/ak_intl_v3/issues/395